docs/components/stepper/index.html

```diff
@@ -1,6 +1,5 @@
 ---
 title: Stepper
```

Comment:
Please add a reference to the `Stepper` component on the `Accordion` page:

https://deploy-preview-675--helix-ui.netlify.app/components/accordion/

Reply:
Changes have been done, please review.